res/controllers/midi-components-0.0.js

```diff
@@ -733,16 +733,53 @@
     });
 
     const JogWheelBasic = function(options) {
+        if (options.deck !== undefined && options.group !== undefined) {
+            console.warn(
+                "options.deck and option.group are both set; " +
+                "options.deck will take priority"
+            );
+        }
+
+        const deck = options.deck;
+        const group = script.deckFromGroup(options.group);
```

Comment:
group is really the deck here now, right?

Reply:
WDYM?

Reply:
well, after assigning to `groupp, `deck == group` should hold (assuming the deck and group is set correctly).

Reply:
Yes. I wasn't sure a deconnection between deck and group was intentional. The previous implementation allowed to set both deck and group with different values and I'm not sure why. Maybe there's a valid use case? Anyway, yes the group could simply be derived from deck, no need to set the variable.

Reply:
Yup, no deck was essentially just supposed to be a cache for `script.deckFromGroup(group)`. If `deck` can be implemented efficiently in terms of `group` (or vice versa), that would be ideal. The previous implementation was just too lazy (by yours truly :sweat_smile:) to care.